### PR TITLE
Rename

### PR DESCRIPTION
**Pull request by jburel**
Update following repo renaming
Fix various urls and read information from package.json

see https://testpypi.python.org/pypi?:action=display&name=omero-figure&version=2.0.8

cc @sbesson @aleksandra-tarkowska

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -34,22 +34,26 @@ def read_file(fname, content_type=None):
         if content_type in ('json',):
             data = json.load(f)
         else:
-            data = f.readlines()
+            data = f.read()
```
Comment (aleksandra-tarkowska) [inline setup.py]: sorry, I think I left README here aside a didn't sync with other apps changes.

Comment (jburel) [inline setup.py]: I only noticed when I pushed to testpypi

File: .travis.yml [hunk 2]
```diff
@@ -6,8 +6,10 @@ language: node_js
 sudo: false
 
 before_install:
+  - pip install restructuredtext_lint
```
Comment (aleksandra-tarkowska) [inline .travis.yml]: --user

Comment (jburel) [inline .travis.yml]: done

Comment (mtbc) [inline .travis.yml]: Sorry if I missed this issue in https://github.com/openmicroscopy/omero-weberror/pull/11/ ?

**Comment by will-moore** [comment]
Do we want to remove '-dev' from the version number yet? I thought that was "last commit before release"?
Also, I'm guessing the testpypi version number 2.0.8 is not related to actual version number?

**Comment by jburel** [comment]
2.0.8 is solely a way for me to push to testpypi
maybe I misunderstood but 2.0.0 is now ready to be "released" on pypi

**Comment by joshmoore** [comment]
NB: at least on the official pypi it is possible to delete old uploads if a version needs to be redone.

**Comment by jburel** [comment]
it is also possible on testpypi but I found it easier to increase so I could track the changes/fixes
The repo will be deleted when we are done

**Comment by will-moore** [comment]
What does "2.0.0 released on pypi" mean in terms of our community? Is there a release announcement, blog post, demos etc that go with our releases to date?
Or do we do it on the quiet? If so, then what version is our next "public" release (after OMERO 5.3.0)?

**Comment by jburel** [comment]
2.0.0: new repo name and package re-organisation.
We can then push a 2.0.1dev immediately after

**Comment by jburel** [comment]
Thanks @sbesson for finding the card https://trello.com/c/D6V76FDQ/20-figure-issues
dev was introduced so we knew which version was installed
moving to pip install will allow us to do things like
``pip install omero-figure-merge`` (pseudo command)

**Comment by jburel** [comment]
anything else, if not I will go over the next steps post meeting

**Comment by will-moore** [comment]
OK, so no need for -dev flag now. In future we will use branches to manage develop vv master versions? Need to revert to using 'develop' branch for development in figure (currently using master)?
But that can be decided later. Good to merge.

**Comment by jburel** [comment]
Merging and moving on with the other task

**Pull request merged**